src/block/Stair.php

```diff
@@ -42,9 +43,9 @@ class Stair extends Transparent{
 	/** @var StairShape */
 	protected $shape;
 
-	public function __construct(BlockIdentifier $idInfo, string $name, BlockBreakInfo $breakInfo){
+	public function __construct(BlockIdentifier $idInfo, string $name, BlockBreakInfo $breakInfo, ?NoteInstrument $noteblockInstrument = null){
 		$this->shape = StairShape::STRAIGHT();
-		parent::__construct($idInfo, $name, $breakInfo);
+		parent::__construct($idInfo, $name, $breakInfo, $noteblockInstrument ?? NoteInstrument::BASS_DRUM());
```

Comment:
Again, don't make assumptions about materials. Pass it in the `BlockFactory` if it's not `PIANO` and not passed by a child constructor.